DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/PromoteToAdminController.cs

```diff
@@ -173,7 +173,7 @@ public IActionResult Index(AdminRolesFormData formData, int delegateId)
                 return new StatusCodeResult(500);
             }
             TempData["IsDelegatePromoted"] = true;
-            return RedirectToAction("Index", "ViewDelegate", new { delegateId });
+            return RedirectToAction("Index", "ViewDelegate", new { delegateId = delegateId, callType = 1 });

```

Comment:
What does callType = 1 mean here? Would an enum make this more understandable? (Genuine question - might be overkill tbh).

Reply:
This is fixed.